Adding Android Sample App

bboothe-branch opened the pull request:
Initial commit of files for Android Sample App

bboothe-branch committed:
Initial Commit

bboothe-branch committed:
Removed Empty Branch Utils Class

bboothe-branch committed:
Update activity_color_block_page.xml

gdeluna-branch reviewed (changes requested):
Added some feedback, lmk if you wanna go over it- we can use the working session or a one off

bboothe-branch committed:
Commit after Gabe Feedback

bboothe-branch committed:
Commit after Gabe Fedback

bboothe-branch committed:
Update Gitignore

Added Gitignore to root folder

bboothe-branch committed:
Removed .idea and .gradle

Removed .idea and .gradle. Modified .gitignore

bboothe-branch committed:
Updates to Codw

Updates to Logs, readability, and some UI improvements

bboothe-branch committed:
Updates to MainActivity and Routing

Fixed Change Branch Badge logic by adding a ViewModel

Updated routing to have the option of arriving on the Read Deep Link Page

Fixed Session Parameters to properly show deep link information on Read Deep Link Page

Specified Android SDK to 5.3.0

bboothe-branch committed:
Update to ViewModel

Made ViewModel change based on the object instead of a number

bboothe-branch committed:
Switch ViewModel to Boolean value
